mimir-build-image/Dockerfile

```
@@ -9,6 +9,10 @@ FROM golang:1.22.5-bookworm
 ARG goproxyValue
 ENV GOPROXY=${goproxyValue}
 ENV SKOPEO_DEPS="libgpgme-dev libassuan-dev libbtrfs-dev libdevmapper-dev pkg-config"
+# Override toolchain directive in go.mod, to ensure the image's Go version is used.
+# Be aware that the official Go Dockerfiles already do this, but let's be explicit.
+# https://github.com/docker-library/golang/issues/472
+RUN go env -w GOTOOLCHAIN=local
```

(This is super nit), but this Dockerfile sets `GOPROXY` and `GOCACHE` as explicit environment variables. Is there any reason to set `GOTOOLCHAIN` via `go env`?

```suggestion
ENV GOTOOLCHAIN=local
```

---

Good point @narqo, as [the official Go image](https://github.com/docker-library/golang/blob/539882fb23e90d31854a51a773accf8731cf0c9d/1.22/bookworm/Dockerfile#L123) also uses `ENV`. I thought they were using `go env -w`.